saul-core/doc/SAULLANGUAGE.md

```diff
@@ -0,0 +1,77 @@
+## Classifiers
```

Comment:
Actually, as I said no information added, but I think we need to keep those on the main `CONCEPTUALSTRUCTURE.md` and the new files have those information again and a extension with more details and more examples later. Is that ok?